### PR TITLE
Dashbugs

### PR DESCRIPTION
**Pull request by Devilla**

### CODE REVIEW

**Comment by GAUTAMRAJU15** [comment]
pLS REVIEW

File: src/components/InstallPixel/InstallPixel.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ const InstallPixel = ({
       </div>
       <div className="float-left custom-width align-install-btn">
         <button type="button" className="btn btn-primary waves-effect copy-btn" onClick={() => { this.buttonDOM.blur(); handlePixelCopy(); }} ref={(buttonDOM) => this.buttonDOM = buttonDOM}>Copy</button>
+        {elastic!==undefined && elastic=='verified'}
```
Comment (ShankyRana) [inline src/components/InstallPixel/InstallPixel.js]: ???

File: src/components/CapturePage/CapturePage.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ class CapturePage extends Component {
   addPageUrl = () => {
     if(this.state.lead.url == '')
       return this.setState({error: 'Please enter a valid path'});
-    if(this.state.lead.url[0]!=='/')
+    if(this.state.displayUrl.url==undefined || this.state.lead.url[0]!=='/')
```
Comment (ShankyRana) [inline src/components/CapturePage/CapturePage.js]: if this.state.displayUrl.url is undefined it should result in error.

File: src/components/DisplayPage/DisplayPage.js [hunk 3]
```diff
@@ -51,18 +53,29 @@ class DisplayPage extends Component {
...
   handleBackState = () => {
-    this.props.setActiveState(3);
+    this.props.setActiveState(4);
   }
 
   addPageUrl = () => {
     if(this.state.displayUrl.url == '')
       return this.setState({error: 'Please enter a valid path'});
-    if(this.state.displayUrl.url[0]!=='/')
+    if(this.state.displayUrl.url==undefined || this.state.displayUrl.url[0]!=='/')
```
Comment (ShankyRana) [inline src/components/DisplayPage/DisplayPage.js]: same with this.

File: src/components/CapturePage/CapturePage.js [hunk 4]
```diff
@@ -58,8 +58,16 @@ class CapturePage extends Component {
 
 
   addPageUrl = () => {
-    if(this.state.lead.url == '')
-      return this.setState({error: 'Please enter a valid path'});
+    if(this.state.displayUrl.url == ''){
+      if(this.state.count<1)
```
Comment (ShankyRana) [inline src/components/CapturePage/CapturePage.js]: count is for?

**Pull request merged**